spec/support/sidekiq.rb

```diff
@@ -0,0 +1,7 @@
+RSpec.configure do |config|
+  config.around sidekiq: true do |example|
+    Sidekiq::Testing.inline! do
+      example.run
+    end
+  end
+end
```

Comment:
This makes tests easier but it might make tests slower since this is gonna trigger all the sidekiq actions, even those which we do not care about the result (a.i. many emails).

In case it really slow down tests, maybe we should use `Sidekiq::Testing.inline!` only withing the tests that we care about the result of the action enqueued in sidekiq.

Reply:
That's how this works - you can tag the test with `sidekiq: true` and it'll run for those tests.

Reply:
this sounds good 👍